docs/2-virtual-machines-containers/2.3-managing-infrastructure.md

```diff
@@ -4,34 +4,25 @@ docs/2-virtual-machines-containers/2.3-managing-infrastructure.md:
   estReadingMinutes: 10
   exercises:
     -
-      name: Jenkins and Artifactory on VMs
-      description: Create VMs from your golden image. On one VM install Jenkins and configure workflow to build Spring’s PetClinic. On the other VM install and configure Artifactory. Configure these VMs so they can communicate and create another workflow that pushes the build artifact from the Jenkins server to Artifactory
+      name: Artifactory on VMs
+      description: Create a VM from your golden image and install and configure Artifactory and deploy an artifact to that VM from a GitHub Action
       estMinutes: 390
       technologies:
       - Vagrant
       - CentOS
-      - Jenkins
+      - GitHub Actions
       - Artifactory
 ---
 
 # Managing Infrastructure
 
-In this section, you will be taking what you've learned about virtual machines to build on top of your Golden Image and deploy servers to create a simple software delivery environment. We will cover the tools used in this section in greater detail later in the Bootcamp but for now here are some basic definitions to help you understand the environment will be creating.
-
-## Jenkins
-
-  Jenkins is a build automation server that manages and controls software delivery processes throughout the entire life cycle. It orchestrates tasks including build, test, package, staging, deployment, static code analysis.
-
-![jenkins image](img2/jenkins.svg ':size=226x312 :class=icon :alt= jenkins image')
-
-- A leading open source continuous integration tool
-- Builds, tests, and deploys software continuously and monitors the execution and status of jobs
-- Trigger jobs manually, on a schedule, or on a code commit
-- Uses Java and Tomcat to do its work
+In this section, you will be learning how to deploy your golden image (and other artifacts) to a centralized location from your build pipeline.
 
 ## Artifactory
 
-  Artifactory is a versioned artifact repository. It is used to store multiple versions of different packages and supports different repository types (Maven, npm, Docker, etc). It is scalable and integrates with lots of other software delivery tools.
+  [Artifactory](https://jfrog.com/artifactory/install/) is a versioned artifact repository. It is used to store multiple versions of different packages and supports different repository types (Maven, npm, Docker, etc). It is scalable and integrates with lots of other software delivery tools.
+
+  ?> Artifactory is not free, but does have a 14 day [free trial](https://jfrog.com/start-free/) you can use. You will want to select the Self-Hosted option.
```

Comment:
Well that is concerning... If only because we ask apprentices to use the artifactory image throughout this exercise. Looks like we should probably look for an alternative FOSS artifact repository, though that is a ticket for another day.

Reply:
Hmm while an attractive idea I think we want to keep it the artifact repository as an external system they have to manage. If only because getting separate containers or instances to communicate is a skill we want them to get experience with